evalai/utils/auth.py

```diff
@@ -76,6 +76,37 @@ def get_user_auth_token():
         sys.exit(1)
 
 
+def validate_user_auth_token_by_profile(auth_token):
+    """
+    Checks tokens availability via server.
+    """
+    url = "{}{}".format(get_host_url(), URLS.profile.value)
+    try:
+        headers = {"Authorization": "Token {}".format(auth_token)}
+        response = requests.post(url, headers=headers)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        if response.status_code in EVALAI_ERROR_CODES:
+            echo(
+                style(
+                    "Failed: Could not make a authentication with this token.",
+                    bold=True,
+                    fg="red"
+                )
+            )
+            sys.exit(1)
+    except requests.exceptions.RequestException:
+        echo(
+            style(
+                "\nCould not establish a connection to EvalAI."
+                " Please check the Host URL.\n",
```

Comment:
@takitsuse 
The error should also display the Host URL not that is not being able to connect to.
For Example, if the host is https://test.evalai.org is CLI is not able to establish connection to it, the error should be:
``` sh
Could not establish a connection to EvalAI. Please check the Host URL: https://test.evalai.org
```